FreeRTOS/Test/CMock/queue/sets/queue_in_set_utest.c

```diff
@@ -75,47 +75,6 @@ int suiteTearDown( int numFailures )
 /* ==============================  Test Cases =============================== */
 
 
-/**
```

Comment:
Removing this test case might be what's causing lower coverage in prvNotifyQueueSetContainer.